tests/testdata/version.txt

```diff
@@ -1 +1 @@
-97a02574e0b54c89017558d8095d1fcec2b64ac4
\ No newline at end of file
+884f7d846bcc145174d019ee881a737c12f584ff
```

Comment:
I don't see this commit in contracts' main commit history. Should we pick a commit from there?

Reply:
Sure, I can use the new version tag here.